modules/proxistoreBidAdapter.md

```diff
@@ -0,0 +1,64 @@
+# Overview
+
+```
+Module Name:  Proxistore Bid Adapter
+Module Type:  Bidder Adapter
+GDPR compliant: true
+```
+
+# Description
+
+Connects any publisher to Proxistore.com's exchange for bids.
+
+The present document is available [online](https://abs.proxistore.com/scripts/proxistore-prebid-adapter.md);
+the companion Javascript file is available [online](https://abs.proxistore.com/scripts/proxistore-prebid-adapter.js) as well. 
+
+# Sample Ad Unit: For Publishers
+
+Parameters:
+* website: __Required__ - Publisher website as registered with Proxistore - __Each publisher must get this value from Proxistore__
+* language: __Required__ - Publisher website language - Must be one of __fr__,__nl__,__es__
+
+Example for a website 'example.com' in French
+
+```
+var adUnits = [
+{
+    sizes: [
```

Comment:
PBJS 3.0 (Coming soon) will remove the support for these "Top Level" Sizes.

Ad Units will now need to place sizes within the appropriate mediaTypes objects.

```javascipt
{
                    code: 'Test-Slot-2',
                    mediaTypes: {
                        banner: {
                            sizes: [[300,250]]
                        }
                    },
                    bids: [
                        {
                            bidder: 'proxistore',
                            params: {
                               website: 'example.com',
                               language: 'fr'
                            }
                        }
                    ]
```

Also note, that an AdUnit requires a `code` and your examples do not show 

The adapter `.md` file is meant for users to be able to plug into their test pages and validate the adapter works correctly.

So I should be able to copy the entire ad unit and paste it into a test page and see valid bids from proxistore.